app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksScreenTest.kt

```diff
@@ -99,7 +99,7 @@ class TasksScreenTest {
     @Test
     fun displayCompletedTask() = runTest {
         repository.apply {
-            createTask("TITLE1", "DESCRIPTION1").also { completeTask(it.id) }
+            createTask("TITLE1", "DESCRIPTION1").also { completeTask(it) }
```

Comment:
Same as above. I suppose there are different opinions about it, but I just don't see a reason to create a redundant lambda. IMO it's best to simply pass the function.